src/OSPSuite.Presentation/Presenters/Charts/CurveSettingsPresenter.cs

```diff
@@ -203,5 +208,32 @@ public void UpdateColorForCurve(Curve curve, Color color)
          var curveDTO = findCurveDTOFor(curve);
          UpdateCurveColor(curveDTO, color);
       }
+
+      public void EditProperties(List<CurveDTO> selectedCurveDTOs)
+      {
+         using (var multiEditorPresenter = _applicationController.Start<ICurveMultiItemEditorPresenter>())
+         {
+            multiEditorPresenter.Show();
```

Comment:
Just encapsulate this behaviour. You are calling 3 presenter methods in a row. 
Call only one and put the logic in the presenter